### PR TITLE
add listen network info

### PR DESCRIPTION
**Pull request by JimYam**

### CODE REVIEW

File: packages/apps-config/src/endpoints/productionRelayKusama.ts [hunk 1]
```diff
@@ -181,6 +181,17 @@ export function createKusama (t: TFunction): EndpointOption {
           OnFinality: 'wss://rpc.api.kico.dico.io'
         }
       },
+      {
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/productionRelayKusama.ts:184]: Ordering is where CI fails here - move this alphabetical.

Comment (JimYam) [reply packages/apps-config/src/endpoints/productionRelayKusama.ts]: I'm so sorry, it is my wrong. I've just modified it.

**Comment by JimYam** [comment]
hey, test is ok. Could you please give us pr for the merger today? thanks!

**Comment by JimYam** [comment]
hey, test is ok. Could you please merge our PR today? thanks!

**Review by jacogr — APPROVED** [review]

**Pull request merged**